Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -163,16 +177,17 @@ public final class AlertManager {
             }
 
             notification.title = NSLocalizedString("Loop Failure", comment: "The notification title for a loop failure")
+            let shouldMuteAlert = alertMuter.shouldMuteAlertIssuedFromNow(failureInterval)
             if isCritical, FeatureFlags.criticalAlertsEnabled {
                 if #available(iOS 15.0, *) {
                     notification.interruptionLevel = .critical
                 }
-                notification.sound = .defaultCritical
+                notification.sound = shouldMuteAlert ? .defaultCriticalSound(withAudioVolume: 0.0) : .defaultCritical
             } else {
                 if #available(iOS 15.0, *) {
                     notification.interruptionLevel = .timeSensitive
                 }
-                notification.sound = .default
+                notification.sound = shouldMuteAlert ? nil : .default
```

Comment:
I think this just sets the sound to use default, not to actually mute the sound.

Reply:
I'll test again, but from initial testing this mutes the sound, but not the vibration.